src/app/(general)/api/superadmin/valid/route.js

```diff
@@ -0,0 +1,14 @@
+import { PrismaClient } from "@prisma/client";
+import { NextResponse } from "next/server";
+
+export async function POST(request) {
+  const prisma = new PrismaClient();
+  const params = await request.json();
+
+  const valid = await prisma.admin.findMany({
+    where: { type: "superadmin" },
```

Comment:
Would be better to fetch specific superadmin by including the email. Fetching all superadmins can be a security risk by checking the fetch response.